google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -365,6 +380,30 @@ public Synchronicity getSynchronicity() {
     return getLogging().getWriteSynchronicity();
   }
 
+  /**
+   * Sets the metadata auto population flag.
+   *
+   * @param synchronicity {@link Synchronicity}
```

Comment:
Seems param comment is not relevant?

Reply:
OMG. Will fix this one :+1:

Reply:
Done. (lines above are marked with `Outdated`)